lib/kitchen/provisioner/chef_zero.rb

```diff
@@ -195,6 +207,23 @@ def prepare_chef_client_zero_rb
         FileUtils.cp(source, File.join(sandbox_path, "chef-client-zero.rb"))
       end
 
+      HANDLER_CODE = <<-EOH
+# Handler to kill the run if any resource is updated
+class UpdatedResources < ::Chef::Handler
```

Comment:
I am not sure to understand what you mean, the code for this handler is already written in the client_no_updated_resources.rb file which is copied from the sandbox.

Do you mean that you want the handler code to be written in a specific file and included in client.rb conf using an include such as:
```ruby
Dir.glob(File.join("/etc/chef", "client.d", "*.rb")).each do |conf|
  Chef::Config.from_file(conf)
end
```